API Blueprint Specification.md

```diff
@@ -30,6 +30,8 @@ Version: 1A8
 ### Section Basics
 + [Metadata section](#def-metadata-section)
 + [API name & overview section](#def-api-name-section)
++ [Authentication section](#def-authentication-section)
```

Comment:
Similarly to Data Structures I would like to move these two sections from "Basics" to "Going Further" – if you are trying to get a grasp on what API Blueprint is the authentication sections are not the first things you should be learning..